karma.conf.js

```diff
@@ -34,7 +34,9 @@ module.exports = function (config) {
       // we use the BDD interface for all of the tests that
       // aren't interface-specific.
       'test/browser-fixtures/bdd.fixture.js',
-      'test/unit/*.spec.js'
+      'test/unit/*.spec.js',
+      'test/browser-unit/*.spec.js',
```

Comment:
Planning on filling `test/browser-unit` with tests for `lib/browser/*.js`, especially any that don't just end up covered by extension of being used in other tested stuff.